docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
+| username                          | Yes          | NONE              | The username of the account that you want to use to load data into StarRocks.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| password                          | Yes          | NONE              | The password of the preceding account.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| sink.version                      | No           | AUTO              | Supported since 1.2.4. Choose which implementation to use. <ul><li>`V1`: use [Stream Load](https://docs.starrocks.io/en-us/latest/loading/StreamLoad) interface to load data. Connectors before 1.2.4 only support this mode. </li> <li>`V2`: use [Transaction Stream Load](https://docs.starrocks.io/en-us/latest/loading/Stream_Load_transaction_interface) interface to load data. It requires StarRocks to be at least version 2.4. Recommends `V2` because it optimizes the memory usage and provide a more stable exactly-once implementation. </li> <li>`AUTO`: if the version of StarRocks supports transaction stream load, will choose `V2` automatically, otherwise choose `V1` </li></ul> |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/9f324354-1424-4eab-b3de-b527c1e36137)
Give the param description first, and then tell the user from which version this param is supported. -> The implementation to use. This parameter is supported from 1.2.4 onwards.